StyleCop.Analyzers/StyleCopTester/Program.cs

```diff
@@ -39,6 +39,10 @@ private static void Main(string[] args)
                     cts.Cancel();
                 };
 
+            // The following allows continuations to be processed by a single thread.
```

Comment:
:arrow_right: At the request of @hmemcpy, I made these corrections during the merge.